benchmark/cuchembm/metrics/embedding.py

```diff
@@ -219,14 +226,14 @@ def _calculate_metric(self, embeddings, fingerprints, estimator, param_dict):
     def calculate(self, estimator, param_dict, **kwargs):
         embeddings = self.encode_many(zero_padded_vals=False, average_tokens=True)
         embeddings = xpy.asarray(embeddings, dtype=xpy.float32)
-        fingerprints = xpy.asarray(self.fingerprint_dataset.data.values, dtype=xpy.float32)
+        fingerprints = xpy.asarray(self.fingerprint_dataset.values, dtype=xpy.float32)
 
         results = self._calculate_metric(embeddings,
                                          fingerprints,
                                          estimator,
                                          param_dict)
         ratio, fingerprint_error, embedding_error, fingerprint_param, embedding_param = results
-        property_name = self.smiles_dataset.properties.columns[0]
+        property_name = self.smiles_properties.columns[0]
```

Comment:
Should probably add a check to ensure there's only one property. And then potentially get rig of the list? can wait for a refactor, if you prefer.

Reply:
Once merged it will get added. I am not sure why it is not appearing in the PR. Please refer the screenshot

![image](https://user-images.githubusercontent.com/1195618/141398930-ddec50ca-b4ff-4623-87aa-42e2f5a0345b.png)